datastruct.erb

```diff
@@ -1 +1,7 @@
-{"disk_sizes": "<%= disk_sizes_meas %>", "disk_usage": "<%= disk_usage_meas %>", "disk_template": "<%= disk_template_meas %>", "num_cpus": "<%= num_cpus_meas %>", "total_ram": "<%= total_ram_meas %>", "active": <%= active_meas %> }
+{"disk_sizes": "<%= disk_sizes_meas %>",
+ "disk_usage": "<%= disk_usage_meas %>",
+ "disk_template": "<%= disk_template_meas %>",
+ "num_cpus": "<%= num_cpus_meas %>",
+ "total_ram": "<%= total_ram_meas %>",
+ "active": <%= active_meas %>
+ }
```

Comment:
I did that to make it easier to read. If you want I can move it to a separate PR but since it was so simple I left it in here.